biztalk/core/step-8-on-premises-configure-the-biztalk-server-application.md

```diff
@@ -91,7 +91,7 @@ In the previous step you created a [!INCLUDE[btsBizTalkServerNoVersion](../inclu
       |**Issuer key**|Specify the issuer key.|
 
       > [!NOTE]
-      >  You can get the values for the Queue URL, ACS URL, issuer name and key from the [Windows Azure CTP Management Portal](https://go.microsoft.com/fwlink/p/?LinkId=202886).
+      >  You can get the values for the Queue URL, ACS URL, issuer name and key from the Windows Azure CTP Management Portal.
```

Comment:
"Windows Azure CTP Management Portal" is a really old name where CTP means "Community Technical Preview". The current name is "Microsoft Azure portal", or "Azure portal":

```suggestion
      >  You can get the values for the Queue URL, ACS URL, issuer name and key from the [Microsoft Azure portal](https://portal.azure.com).
```